.github/workflows/security.yml

```
@@ -0,0 +1,112 @@
+      - name: Run Nancy
+        uses: sonatypecommunity/nancy-github-action@main
+        with:
+          nancyCommand: sleuth --exclude-vulnerability CVE-2021-1234
+
+      
+
+  # Container Scanning (if Dockerfile exists)
+  container:
+    name: Container Scan
+    runs-on: ubuntu-latest
+    if: ${{ hashFiles('Dockerfile') != '' }}
```

actionlint (1.7.11)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)